subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -680,19 +680,13 @@ u32_t radio_tmr_start(u8_t trx, u32_t ticks_start, u32_t remainder)
 	NRF_PPI->CH[1].TEP = (u32_t)&(NRF_TIMER0->TASKS_START);
 	NRF_PPI->CHENSET = PPI_CHEN_CH1_Msk;
 
+	NRF_PPI->CH[0].EEP = (u32_t)&(NRF_TIMER0->EVENTS_COMPARE[0]);
 	if (trx) {
-		NRF_PPI->CH[0].EEP =
-			(u32_t)&(NRF_TIMER0->EVENTS_COMPARE[0]);
-		NRF_PPI->CH[0].TEP =
-			(u32_t)&(NRF_RADIO->TASKS_TXEN);
-		NRF_PPI->CHENSET = PPI_CHEN_CH0_Msk;
+		NRF_PPI->CH[0].TEP = (u32_t)&(NRF_RADIO->TASKS_TXEN);
 	} else {
-		NRF_PPI->CH[0].EEP =
-			(u32_t)&(NRF_TIMER0->EVENTS_COMPARE[0]);
-		NRF_PPI->CH[0].TEP =
-			(u32_t)&(NRF_RADIO->TASKS_RXEN);
-		NRF_PPI->CHENSET = PPI_CHEN_CH0_Msk;
+		NRF_PPI->CH[0].TEP = (u32_t)&(NRF_RADIO->TASKS_RXEN);
 	}
+	NRF_PPI->CHENSET = PPI_CHEN_CH0_Msk;
```

Comment:
add an empty line after the }

Reply:
Should just use a conditional assignment statement instead for my changes?